components/stepper/stories/template.js

```diff
@@ -28,19 +32,23 @@ export const Template = ({
       'is-invalid': isInvalid,
       'is-disabled': isDisabled,
       ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
-    })} id=${ifDefined(id)}>
+    })} id=${ifDefined(id)}
+    style=${ifDefined(styleMap(style))}
+    >
       ${Textfield({
         ...globals,
         placeholder: "100",
         min: "-2",
         max: "2",
         step: "0.5",
+        isDisabled,
+        isQuiet,
         customClasses: [`${rootClass}-textfield`],
         customInputClasses: [`${rootClass}-input`],
       })}
       <span class="${rootClass}-buttons">
-        ${ActionButton({ ...globals, customClasses: [`${rootClass}-stepUp`], iconName: 'ChevronUp75', isDisabled })}
-        ${ActionButton({ ...globals, customClasses: [`${rootClass}-stepDown`], iconName: 'ChevronDown75', isDisabled })}
+        ${ActionButton({ ...globals, customClasses: [`${rootClass}-stepUp`], icon: 'ChevronUp75', isDisabled, isQuiet })}
+        ${ActionButton({ ...globals, customClasses: [`${rootClass}-stepDown`], icon: 'ChevronDown75', isDisabled, isQuiet })}
```

Comment:
```suggestion
        ${ActionButton({ ...globals, customClasses: [`${rootClass}-stepUp`], iconName: 'ChevronUp75', isDisabled, isQuiet })}
        ${ActionButton({ ...globals, customClasses: [`${rootClass}-stepDown`], iconName: 'ChevronDown75', isDisabled, isQuiet })}
```

This reverted the update to iconName that ActionButton needs.

Reply:
I must not have pulled in your changes or something because I had that and it wasn't working. Fixing it now!